### PR TITLE
Update with changed site URLs and GitHub Actions file

### LINKED ISSUE #22: Update for new URLs and deployment process
Review documentation and update to: 
1. Reflect changing URLs for public/private sites
2. Better explain deployment process

### PR DESCRIPTION
**Pull request by HaSistrunk**
Fixes #22 
Updates documentation to include new public/internal site URLs and use of GitHub Actions to trigger deployment. 

Includes some minor language edits and cleanup.

### CODE REVIEW

**Review by kemartin2015 — APPROVED** [review]
These changes look good to me, @HaSistrunk!

File: add-content.md [hunk 1]
```diff
@@ -10,19 +10,20 @@ Prior to adding or editing content, all additions and changes must be approved a
...
-7. Once a pull request is reviewed and approved, merge the branch into the `base` branch where it will be incorporated into docs.rockarch.org.
+7. Once a pull request is reviewed and approved, merge the branch into the `base` branch where it will be incorporated into the site.
 
 ## Required Files
 
 * [ ] index.md
 * [ ] README.md
 * [ ] LICENSE.md
 * [ ] \_config.yml
+* [ ] publish_sns.yml
```
Comment (helrond) [inline add-content.md]: Worth using the full file path here?
```suggestion
* [ ] /.github/workflows/publish_sns.yml
```

File: add-content.md [hunk 2]
```diff
@@ -10,19 +10,20 @@ Prior to adding or editing content, all additions and changes must be approved a
...
-4. Review the changes on the [RAC development site](docs.dev.rockarch.org).
+4. Review the changes on the [RAC development site](https://docs-internal.dev.rockarch.org/).
 5. Submit a pull request to merge `development` to the `base` branch of the GitHub repo.
 6. As per the [Content Approval Policy](https://docs.rockarch.org/docs-policy/), each pull request will be reviewed by the relevant Assistant Director and the Director of Archives, as necessary. If a determination is made to either convert documentation from public to private, or to remove it completely, the Assistant Director for Digital Strategies or any other RAC employee with administrative access to the RAC GitHub account should be notified to enable this action.
-7. Once a pull request is reviewed and approved, merge the branch into the `base` branch where it will be incorporated into docs.rockarch.org.
+7. Once a pull request is reviewed and approved, merge the branch into the `base` branch where it will be incorporated into the site.
 
 ## Required Files
```
Comment (helrond) [inline add-content.md:18]: The checkboxes below look kind of weird when combined with the bullet list. Should we just remove them?

**Review by helrond — APPROVED** [review]

**Review by daremiyo — APPROVED** [review]
Looks good!

**Pull request merged**